charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml

```diff
@@ -171,6 +171,11 @@ data:
         Parser              kubeapiserverParser
         Reserve_Data        True
 
+    [FILTER]
```

Comment:
Hm, we don't use this construct (`Record type user`) for other component logs that we expose to end-users (controller-manager, scheduler) - how does it work for them?

Reply:
Hm. you are right, as I see now we use only predefined dashboards.